packages/jest/jest.config.ts

```diff
@@ -1,3 +1,4 @@
+const j = require('./tsconfig.json');
```

Comment:
What is the purpose of this import?

Reply:
oops, was testing something and forgot to remove this.